migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/DefaultMigrationTarget.java

```diff
@@ -674,7 +674,7 @@ public void savePools(final Collection<Pool> pools) throws IOException {
 
   private void resolvePoolables(Pool pool) throws IOException {
     PoolableElementViewService svc = serviceManager.getPoolableElementViewService();
-    for (PoolDilution pd : pool.getPoolDilutions()) {
+    for (PoolElement pd : pool.getPoolContents()) {
```

Comment:
'pd' no longer makes sense as a variable name, here and many other places

Reply:
Renamed everywhere I found it